.github/workflows/sycl-windows-run-tests.yml

```diff
@@ -18,6 +18,16 @@ on:
       ref:
         type: string
         required: False
+      build_ref:
```

Comment:
can we add a `description` explaining the difference between `build_ref` and `ref`? thx

Reply:
Is it better? [649afb6](https://github.com/intel/llvm/pull/16621/commits/649afb691f5efa17b18c4c98618fb0cca74c6b5d)

Reply:
still unclear to me `build_ref` vs `ref`

Reply:
Me too:D it's not me who created this. Probably we can just use ref in both cases.

Reply:
Or, I don't know the original intent to have `ref` and `build_ref`, but may be it's to have a possibility to download the `devops` dir and the rest repo from different branches.

Reply:
as far as i know theres only a single ref that checks out the source branch for tests to run, if its not too much work maybe we try combining the two

Reply:
I'm afraid even we'd do some digging/archeology on git history, the result would it just happened without well-defined purpose behind it. Reviewing all `*ref` and removing/unifying unnecessary sounds like the right path forward.

Reply:
Ready for merge

> cool, ping me again when youre ready for review/merge